functions/start-friend-call/src/main.js

```
@@ -20,21 +17,27 @@ module.exports = async function ({ req, res, log, error }) {
+      {
+        message: "Failed to send notification",
+        error: err.message,
+      },
+      500
+    );
+  }
 
   return res.json({
-    message: 'Notification sent'
+    message: "Notification sent",
   });
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)